tests/unit/test_crud.py

```diff
@@ -77,6 +78,116 @@ def test_get_contact_by_email_id_miss(dbsession):
     assert contact is None
 
 
+def test_get_bulk_contacts_some_after_higher_limit(
```

Comment:
Not sure if I used the StatementWatcher here correctly. Tried to follow the patterns y'all had in place for pytest fixtures and dealing with db queries.

Reply:
yep, this is how it is supposed to be used, as a context manager when you want to confirm there weren't unexpected SQL queries

Reply:
I am using ``watcher`` instead of ``sw`` as the variable name, because pylint is complaining about short names. For the lint check, we're telling pylint to check unit tests, but my editor is running it on every save.